etsin_finder/frontend/js/components/general/loggedInUser.jsx

```diff
@@ -7,14 +7,16 @@ import { faUser } from '@fortawesome/free-solid-svg-icons'
 
 const LoggedInUser = ({ Stores }) => {
   const { user, userLogged } = Stores.Auth
-  const { firstName } = user
+  const { firstName, lastName } = user
```

Comment:
I think we should only show the first name. We initially tried to show both names but it took too much space. Or if it's possible that only one of them is defined, maybe show `firstName || lastName`.